runtime/onert/core/include/ir/operation/Reshape.h

```diff
@@ -37,12 +37,22 @@ class Reshape : public Operation
     SHAPE = 1
   };
 
+  struct Param
```

Comment:
IMO, it is strange because `new_shape` is duplicated as input and parameter.

Reply:
@periannath Could you explain me reason of duplication `new_shape`?

Reply:
@hseok-oh It is specified in TFLite specification as I described in #2393. I and @ragmani think `new_shape` param is defined for constant shape and `shape` tensor is defined for dynamic shape. However, it seems `shape` tensor is used for both cases. I've uploaded this PR to support `reshape` operator without `shape` tensor.

Reply:
I've checked tensorflow lite kernel. Tensorflow lite check number of inputs and use parameter if node has only one input, otherwise use second input. IMO we should use similar implementation because we have IR definition based on tensorflow lite schema.